pytripgui/config_vc/config_cont.py

```diff
@@ -0,0 +1,31 @@
+import logging
+logger = logging.getLogger(__name__)
+
+
+class ConfigController(object):
+    def __init__(self, model, view):
+        self.model = model
+        self.view = view
+        self.user_clicked_save = False
+
+    def set_view_from_model(self):
```

Comment:
In previous PR we suggested to avoid using setters and getters, as this code is written from scratch, could you please follow same suggestion ?